src/defaultTo.js

```diff
@@ -19,7 +19,8 @@ var _curry2 = require('./internal/_curry2');
  *      defaultTo42(null);  //=> 42
  *      defaultTo42(undefined);  //=> 42
  *      defaultTo42('Ramda');  //=> 'Ramda'
+ *      defaultTo42(parseInt('string')); //=> 42
  */
 module.exports = _curry2(function defaultTo(d, v) {
-  return v == null ? d : v;
+  return v == null || v !== v ? d : v;
```

Comment:
Yes. I did consider that. But it seems like a bigger overhead to me. Then I'd rather have an internal `_isNaN` and make `R.identical` use it as well. I guess it depends on how familiar the `v !== v` is to people.